account_asset_management/models/__init__.py

```diff
@@ -0,0 +1,9 @@
+from . import account_account
+from . import account_asset
+from . import account_asset_profile
+from . import account_asset_line
+from . import account_asset_recompute_trigger
+from . import account_invoice
+from . import account_move
+#from . import date_range
```

Comment:
we should replace this import by "from . import account_fiscal_year", rename the date_range.py to account_fiscal_year.py and adjust the logic for table recompute when creating/changing fiscal year duration.